mmv1/products/networkconnectivity/Spoke.yaml

```diff
@@ -329,3 +329,9 @@ properties:
       - 'CREATING'
       - 'ACTIVE'
       - 'DELETING'
+  - name: 'stateReason'
```

Comment:
The public API documentation suggestions that stateReason does not exist.

[Perhaps reason is what you are looking for?](https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1beta/projects.locations.spokes#:~:text=reasons%5B%5D,the%20INACTIVE%20state.)